rightscale/resource_instance.go

```diff
@@ -21,7 +31,9 @@ func resourceInstance() *schema.Resource {
 			"associate_public_ip_address": &schema.Schema{
 				Description: "Specify whether or not you want a public IP assigned when this Instance is launched. Only applies to Network-enabled Instances. If this is not specified, it will default to true.",
 				Type:        schema.TypeBool,
-				Optional:    true,
+				Default:     true,
+				//ValidateFunc: validation.StringInSlice([]string{"true", "false"}, false),
```

Comment:
Should we delete this line or also add a comment like we have done further down? ("Uncomment validation when tf adds supported...")